stingray/utils.py

```diff
@@ -13,6 +15,11 @@ def jit(fun):
         return fun
 
 
+class UnrecognizedMethod(Exception):
+    pass
+
+
+
```

Comment:
The exception is raised by public util's function - so it might be executed everywhere (including other projects).
If somebody would like to handle exception he'll need to write
```
try:
   do_my_stuff_with_util_function()
except:
   handle_expcetion_even_is_not_particular_util_exception_we_would_like_to_handle()
```
So I think custom exception is always a good idea